portfolio/src/main/java/com/google/sps/servlets/LikeDataServlet.java

```diff
@@ -19,37 +19,37 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
-import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that likes a single comment. */
-@WebServlet("/like-data")
+@WebServlet("/likes-management")
```

Comment:
This is another method that affects comments as a resource, so maybe this could be something like `/comments/like`.

Reply:
Sure. I will modify this in the next commit.